src/main/java/ReviewResponse.java

```diff
@@ -2,6 +2,8 @@
 import lombok.Getter;
 import lombok.ToString;
 
+import java.util.List;
+
 @ToString
 @Data
 public class ReviewResponse {
```

Comment:
Directly exposing a setter for 'comment' field without any validation or logic, violating Encapsulation principle. (Severity: medium)